pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -83,11 +83,26 @@ func GetAuditPolicy(audit configv1.Audit) (*auditv1.Policy, error) {
 	p := basePolicy.DeepCopy()
 	p.Name = string(audit.Profile)
 
-	extraRules, ok := profileRules[audit.Profile]
+	for _, cr := range audit.CustomRules {
+		profileRules, ok := profileRules[audit.Profile]
```

Comment:
shouldn't you get the profile from the custom rule?

like:
`profileRules, ok := profileRules[cr.Profile]`

Reply:
yep